examples/secure_p2p_connection.rs

```diff
@@ -393,6 +402,16 @@ fn main() {
                                     peer_id,
                                 );
                             }
+                            crust::Event::ListenerStarted(port) => {
+                                println!("\nListener started on port {}", port);
+                                if !test_extreach {
+                                    println!("Test for External Reachability is disabled");
+                                    let service_unlocked = unwrap!(service.lock());
```

Comment:
Reads strange that you are getting something called unlocked by calling something called `lock()` - maybe call it service_guard or something ?

Reply:
Hmmm, yeah, that sounds better. I'll change it.